O/oneTBB/build_tarballs.jl

```diff
@@ -30,6 +30,11 @@ if [[ ${target} == i686-linux-musl* ]]; then
     atomic_patch -p1 "${WORKSPACE}/srcdir/patches/i686-musl.patch"
 fi
 
+if [[ "${target}" == *-freebsd* ]]; then
+    # Follow oneTBB makefile fix for missing symbols error: https://cgit.freebsd.org/ports/commit/?id=3677983542cc09a0e5f085e463a895e9e4dce9aa
+    export LDFLAGS="-Wl,--undefined-version"
```

Comment:
I don't know whether there's an existing `LDFLAGS` defined (I strongly suspect there isn't) but you could do this just in case:
```suggestion
    export LDFLAGS="${LDFLAGS} -Wl,--undefined-version"
```